pkg/gatherers/clusterconfig/gather_cluster_storageclass.go

```diff
@@ -0,0 +1,70 @@
+package clusterconfig
```

Comment:
We usually use the word `cluster` for "singleton" resources with the same name. This is not the case so it would be better to call it `gather_storageclass.go` or `gather_storageclasses.go`

Reply:
Ops! True, I was moved by the fact that `StorageClasses` are not namespaced resources. Fixed.